lrmd/lrmd.c

```diff
@@ -1321,7 +1322,25 @@ lrmd_rsc_execute_service_lib(lrmd_rsc_t * rsc, lrmd_cmd_t * cmd)
         }
     }
 
-    if (cmd->isolation_wrapper) {
+    if (safe_str_eq(rsc->class, "alert")) {
```

Comment:
Hi Ken,

Thank you for comment.

I forgot a definition of the fixed number.
I add a new definition and make modifications.